draft-ietf-tls-tls13.md

```diff
@@ -4319,6 +4322,25 @@ will receive various TLS 1.x versions in this field, however its value
 MUST always be ignored.
 
 
+## Zero-RTT backwards compatibility
+
+A TLS 1.3 ClientHello with 0-RTT data is not compatible with older servers.
```

Comment:
I think it would be helpful to explain what the problem is here. It's not the ClientHello but the need to skip past the ciphertext.